android/app/src/main/java/com/reactnativenavigation/activities/BaseReactActivity.java

```diff
@@ -355,6 +389,20 @@ public void invokeDefaultOnBackPressed() {
         super.onBackPressed();
     }
 
+    protected void setupDrawer(Screen screen, Drawer drawer, int drawerFrameId, int drawerLayoutId) {
```

Comment:
`setupDrawer` is taking in the layout IDs here since I'm not sure there's a way to get those from the base activity.  I wasn't able to when initially refactoring.